src/containers/App/tests/index.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {App, mapStateToProps} from '../index';
+import renderer from 'react-test-renderer';
+import {MemoryRouter} from 'react-router';
+
+describe('<App />', () => {
+  it('Expect to be render correctly', () => {
+    const props = {
+      buildInformation: {
+        deploymentEnvironment: 'test',
+        frontEndBuldNumber: 123,
+      },
+    };
+    const wrapper = renderer.create(
+      <MemoryRouter>
+        <App {...props} />
+      </MemoryRouter>
+    );
+    expect(wrapper).toMatchSnapshot();
+  });
+  describe('mapStateToProps', () => {
+    it('should return app buildinformation object', () => {
+      expect(
+        mapStateToProps({app: {buildInformation: {}}}).buildInformation
+      ).toEqual({});
```

Comment:
Seems like it's not really testing anything.

Reply:
I just gave an example on how to test the mapstatetoprops function. it should be testing if it contains the values fetched by the component.

Reply:
In line 24, buildInformation has an empty object. Then we expect an empty object in line 25. I think I would be happier with this test if we could expect something that doesn't look like a default value... maybe something like `{ test: 1234 }`. The problem with using values that are also sometimes default values is that it could provide a false positive in the test.